cmd/system-probe/probe.go

```diff
@@ -63,16 +64,13 @@ func CreateSystemProbe(cfg *config.AgentConfig) (*SystemProbe, error) {
 
 // Run makes available the HTTP endpoint for network collection
 func (nt *SystemProbe) Run() {
-	httpMux := http.DefaultServeMux
-
-	// If profiling is disabled, then we should overwrite handlers for the pprof endpoints
-	// that were registered in init():
-	// https://github.com/golang/go/blob/5bd88b0/src/net/http/pprof/pprof.go#L72-L78
-	// We can only do this by creating a new HTTP Mux that does not have these endpoints handled
```

Comment:
Don't erase this comment completely. There's useful context here that can be kept.